modules/schematics/src/store/index.spec.ts

```diff
@@ -215,4 +215,31 @@ describe('Store Schematic', () => {
     );
     expect(content).toMatch(/export interface FeatureState {/);
   });
+
+  it('should fail if a feature state name is not specified', () => {
+    const options = {
+      ...defaultOptions,
+      name: undefined,
+      root: false,
+    };
+
+    let thrownError: Error | null = null;
+    try {
+      schematicRunner.runSchematic('store', options, appTree);
+    } catch (err) {
+      thrownError = err;
+    }
+    expect(thrownError).toBeDefined();
```

Comment:
```suggestion
    expect(() => {
      schematicRunner.runSchematic('store', options, appTree);
    }).toThrowError('Please provide a name for the feature state');
```

Reply:
Did this for my Test now. There is another Test "should fail if specified module does not exist" which use also the `try catch` - should i change this also?

Reply:
Not in the PR, you can open up a new PR if you would like to